### ISSUE TITLE
Tags are not saving while running save command

### ISSUE THREAD

**Issue by ssiddique-info** [cerner/splunk-pickaxe#29]
Installed the latest version (2.8.0). Configured and ran the command "pickaxe save dev".
It pulls everything except tags. I double checked the existence of tags in the configured app and are publicly shared.

**Comment by bbaugher** [comment]
Yes looks like this was [intentional](https://github.com/cerner/splunk-pickaxe/blob/master/lib/splunk/pickaxe/client.rb#L48) stating there wasn't a way to list the existing tags. Though I seem to have found a list tags API [here](https://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTknowledge#search.2Ftags). 

I wouldn't think it would be difficult but I haven't been working on this project besides releasing, reviewing and maintaining. I'd appreciate a PR if you are able
